services/booking-api/helpers/booking.js

```diff
@@ -0,0 +1,41 @@
+import * as request from '../../../libs/request';
+import params from '../../../libs/params';
+import config from '../../../config';
+
+const PATH = {
```

Comment:
Maybe we could name this URI_RESOURCE?
https://restfulapi.net/resource-naming/

Reply:
Sounds like a good suggestion